content/css/concepts/display/terms/float/float.md

```diff
@@ -36,9 +36,13 @@ Making an `img` element float to the left of its container.
 .container {
   height: 200px;
   width: 200px;
+  border: 5px dashed #3a10e5;
 }
 
 .container img {
   float: left;
 }
 ```
+This results in the following output:
```

Comment:
```suggestion

This results in the following output:
```
always add a white space after paragraphs of code snippets

Reply:
Got it. Thank you